README.md

```diff
@@ -8,7 +8,7 @@
 [![API Reference](https://img.shields.io/badge/API%20Reference-blue)](https://pkg.go.dev/github.com/ssvlabs/ssv)
 ![Github Actions](https://github.com/ssvlabs/ssv/actions/workflows/unit-test.yml/badge.svg?branch=main)
 ![Github Actions](https://github.com/ssvlabs/ssv/actions/workflows/lint.yml/badge.svg?branch=main)
-![Test Coverage](./docs/resources/cov-badge.svg)
+[![Coverage Status](https://codecov.io/gh/ssvlabs/ssv/branch/main/graph/badge.svg)](https://codecov.io/gh/ssvlabs/ssv)
```

Comment:
Following the link gives a report on the first alphabetical branch. I think we need to filter the `main` branch

```suggestion
[![Coverage Status](https://codecov.io/gh/ssvlabs/ssv/branch/main/graph/badge.svg)](https://codecov.io/gh/ssvlabs/ssv/tree/main)
```